docs/gds-supported-platform/custom-domains.md

```diff
@@ -0,0 +1,36 @@
+# Adding a custom domain to your GSP cluster
+
+You may wish to host your GSP apps on a custom domain.  This guide
+explains how it works.
+
+## Assumptions
+
+We assume that:
+
+ - you want GSP to automatically provision TLS certificates (rather
+   than providing your own)
+ 
+## Telling the cluster about the domain
+
+To tell a cluster that you want it to manage a particular domain, add
+the name to the `extra-zones` value in the cluster-config repository.
+For example:
+
+    extra-zones: ["my-domain.example.com"]
+
+## Delegating the domain to the cluster
+
+Once you have told the cluster about your custom domain, you need to
+delegate the domain to the cluster.  Currently, the way you do this is
+to go into the AWS console to discover the appropriate NS records for
```

Comment:
So you have to have created the Route 53 hosted zone?

Reply:
Actually reading further down it sounds like setting extra-zones does this, we should probably document that updating that key will have created the hosted zone to be used here.